src/lib/wdi5-bridge.ts

```diff
@@ -347,17 +347,28 @@ export async function _addWdi5Commands(browserInstance: WebdriverIO.Browser) {
     if (!browserInstance.asControl) {
         browserInstance.asControl = function (ui5ControlSelector) {
             const asyncMethods = ["then", "catch", "finally"]
+            const functionQueue = []
             function makeFluent(target) {
                 const promise = Promise.resolve(target)
                 const handler = {
                     get(_, prop) {
+                        functionQueue.push(prop)
                         return asyncMethods.includes(prop)
                             ? (...boundArgs) => makeFluent(promise[prop](...boundArgs))
                             : makeFluent(promise.then((object) => object[prop]))
                     },
                     apply(_, thisArg, boundArgs) {
                         return makeFluent(
-                            promise.then((targetFunction) => Reflect.apply(targetFunction, thisArg, boundArgs))
+                            // When "targetFunction" is empty we can assume that the ui5 control was not found
+                            promise.then((targetFunction) =>
+                                targetFunction
+                                    ? Reflect.apply(targetFunction, thisArg, boundArgs)
+                                    : Logger.error(
+                                          `Can not call "${functionQueue.filter(
```

Comment:
shouldn't `targetFunction` here be sufficient instead of `filter`ing the `functionQueue`? 
And btw: where does filter criteria `name` come from?

Reply:
In the `functionQueue` will be also the `asyncMethods` stored as defined [here](https://github.com/ui5-community/wdi5/pull/309/files#diff-f66bbe33142ab78f9d4f0cb35c417467ba90cbd9522901d86f7655afc2a374f1R349):
```javascript
const asyncMethods = ["then", "catch", "finally"]
```

If I don't filter out these methods the error message looks something like this:
```zsh
[0-0] [wdi5] Can not call "getText,then", because control could not be found
```
or like this
```zsh
[0-0] [wdi5] Can not call "getParent,getVisible,then", because control could not be found
```

`name` is just the function name of the `asyncMethods`. Maybe I can find a better naming for that

Reply:
nah, that's good → thx for the explanation!